static/app/components/profiling/ProfilingOnboarding/util.ts

```diff
@@ -35,8 +35,12 @@ const platformToDocsPlatform: Record<
   'python-tornado': 'python',
 };
 
-export function isProfilingSupportedForProject(project: Project): boolean {
-  return !!(project.platform && platformToDocsPlatform[project.platform]);
+export function isProfilingSupportedOrProjectHasProfiles(project: Project): boolean {
+  return !!(
+    (project.platform && platformToDocsPlatform[project.platform]) ||
+    // if they somehow managed to send profiles already, then profiling is supported
```

Comment:
would be nice to elaborate here that this happens because people can set the wrong platform for the project

Reply:
Ah yes, good call